src_compressible/flux.cpp

```diff
@@ -794,19 +1204,54 @@ void calculateFlux(const MultiFab& cons_in, const MultiFab& prim_in,
                 +  divxp*(prim(i-1,j,k,1)+prim(i,j,k,1))
```

Comment:
General comment, what is the point of divxp, divyp, and divzp if they are always zero?

Reply:
I think, generally, these terms can be non-zero for non-zero bulk viscosity. It is hard-coded here to be zero, but need not be.